pkg/serializer/serializer.go

```diff
@@ -485,3 +477,49 @@ func (s *Serializer) SendContainerLifecycleEvent(msgs []ContainerLifecycleMessag
 
 	return nil
 }
+
+// SendOrchestratorManifests serializes & send orchestrator manifest payloads
+func (s *Serializer) SendOrchestratorManifests(msgs []ProcessMessageBody, hostName, clusterID string) error {
+	if s.orchestratorForwarder == nil {
+		return errors.New("orchestrator forwarder is not setup")
+	}
+	for _, m := range msgs {
+		payloads, extraHeaders, err := getOrchestratorPayloads(m, hostName, clusterID, true)
+		if err != nil {
+			log.Errorf("Unable to encode message: %s", err)
+			continue
+		}
+
+		responses, err := s.orchestratorForwarder.SubmitOrchestratorManifests(payloads, extraHeaders)
+		if err != nil {
+			return log.Errorf("Unable to submit payload: %s", err)
+		}
+
+		// Consume the responses so that writers to the channel do not become blocked
+		// we don't need the bodies here though
+		for range responses {
+
+		}
+	}
+	return nil
+}
+
+func getOrchestratorPayloads(msg ProcessMessageBody, hostName, clusterID string, isManifest bool) (forwarder.Payloads, http.Header, error) {
```

Comment:
nit: imo its more about generating/making etc. instead of getting

Reply:
renamed to `makeOrchestratorPayloads`